downstream/modules/platform/con-operator-upgrade-prereq.adoc

```diff
@@ -0,0 +1,9 @@
+[id="operator-upgrade-prereq_{context}"]
+
+= Prerequisites
+
+
+[role="_abstract"]
+To upgrade to a newer version of {OperatorPlatform}, you must have the following:
+
+* TBD
```

Comment:
Potential things to include:
* It is recommended to  a backup by creating AutomationControllerBackup and AutomationHubBackup objects.  (link to backup/restore docs when complete)
* If this is a minor/major version, we also recommend reading the release notes for new AAP version you are upgrading to (and any intermediate versions).

Reply:
Suggested changes complete.